src/System.Security.Cryptography.Cng/src/Internal/Cryptography/Helpers.cs

```diff
@@ -3,8 +3,6 @@
 // See the LICENSE file in the project root for more information.
 
 using System;
-using System.Diagnostics;
-using System.Globalization;
```

Comment:
Since all this edit is doing is removing some unused usings and trailing blank lines, maybe this file should be reverted.